### PR TITLE
fix: #2391 Browse app page scrolling is disabled when not over app list

### PR DESCRIPTION
**Pull request by nphivu414**
# Pull request checklist

## Does this close any currently open issues?

fixes: #2391 

**Please check if your PR fulfills the following requirements:**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #2391 

## What is the new behavior?

- Browse app page scrolling is disabled when not over app list

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by nphivu414** [comment]
Hi @willmcvay  @tanphamhaiduong 

The root cause is the App List section’s height is currently set to a 100vh for infinite scrolling to work, and it makes the main container understand that the total height of its child is only 100vh, that's the reason why if we place the cursor over the main container, we can’t scroll more.

Another problem I found is that the infinite scrolling component only listen to scroll event of App List component, not the main container, so the infinite scroll won't work if we scroll the main container

So I decide to use another lib for infinite scrolling, it's more lightweight, maintained better and can fixe all issues we have.

File: packages/marketplace/src/components/pages/apps/coming-soon.tsx [hunk 1]
```diff
@@ -26,19 +30,39 @@ const comingSoonImagesList = [
...
+  setCommingSoonAppSectionHeight: React.Dispatch<React.SetStateAction<number>> | undefined,
+) => {
+  return () => {
+    if (setCommingSoonAppSectionHeight) {
+      setCommingSoonAppSectionHeight(containerHeight)
+    }
+  }
+}
+
+const ComingSoonApps: React.FC<CommingSoonAppsProps> = ({ setCommingSoonAppSectionHeight }) => {
```
Comment (willmcvay) [inline packages/marketplace/src/components/pages/apps/coming-soon.tsx:44]: Is it worth wrapping either the component or the handleSetCommingSoonAppSectionHeight func in a React.memo / useMemo so we're not calculating the height constantly?

File: packages/marketplace/package.json [hunk 2]
```diff
@@ -25,18 +25,18 @@
     "prepublish": "npm run snyk-protect"
   },
   "dependencies": {
+    "@reapit/connect-session": "1.0.0-alpha.1",
```
Comment (willmcvay) [inline packages/marketplace/package.json:28]: Think we should remove this, marketplace is already running on v2.0.0 code and has not yet been released

**Review by willmcvay — COMMENTED** [review]
Just a few comments

**Review by vuhuucuong — APPROVED** [review]

**Pull request merged**